public/maintenance.html

```diff
@@ -87,8 +87,8 @@
   <body>
     <div class="dialog">
       <div>
-        <h1>Site is currently down for maintenance.</h1>
-        <p>We are making some changes and will be back shortly, so please try again soon.</p>
+        <h1>Site is currently down for <%= reason ? reason : "maintenance" %> as of <%= Time.now.strftime("%H:%M %Z") %>.</h1>
+        <p>We are making some changes and will be back <%= deadline ? deadline : "shortly" %>, so please try again soon.</p>
```

Comment:
so to deploy with a custom reason/until you do `REASON="version update" UNTIL="tomorrow" cap staging deploy`